content/docs/enterprise/external-data.mdx

```diff
@@ -4,6 +4,11 @@ sidebar_label: External Data
 description: Extend your authorization policies with data from external sources.
 ---
 
+If you believe decisions should be informed by data, then security decisions should be informed by **all** the data. Today, most applications are limited to using user identity when making access decisions — which leaves systems blind to the multi-faceted forms of attack vectors being exploited in the threat landscape. It turns out an access control system is only as good as the data used in policy decisions.
+
+For more information on how context enriches access decisions, read our [blog post here.](https://www.pomerium.com/blog/context-drives-full-access-decision-making/)
+
+
```

Comment:
```suggestion
```

Reply:
@cmo-pomerium it's your own change, go ahead and commit it.